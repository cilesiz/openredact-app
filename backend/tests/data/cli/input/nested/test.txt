Meine E-Mail Adresse lautet test@mail.de.
Sie ist in Deutschland registriert.
