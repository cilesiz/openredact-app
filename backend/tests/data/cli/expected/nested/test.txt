Meine E-Mail Adresse lautet Email 1.
Sie ist in XXXXXXXXXXX registriert.
